cli/test/README.md

```diff
@@ -441,23 +441,31 @@ This document includes CLI commands and the expected CLI output in order to help
 |`fastddsqosprof file.xml set participant[profile].use_builtin_transports arg1 arg2`|`ERROR: set command for Participant <use_builtin_transports> expects 1 arguments and received 2`|`PARTICIPANT_USE_BUILTIN_TRANSPORTS_USAGE`|
 |`fastddsqosprof file.xml set participant[profile].use_builtin_transports true`|N/A|N/A|
 |`fastddsqosprof file.xml set participant[profile].use_builtin_transports false`|N/A|N/A|
-|`fastddsqosprof file.xml set participant[profile].use_builtin_transports other`|`ERROR: value 'other' is invalid boolean`|N/A|
+|`fastddsqosprof file.xml set participant[profile].use_builtin_transports other`|`Fast DDS QoS Profiles Manager exception caught: value 'other' is invalid boolean`|N/A|
 |`fastddsqosprof file.xml set participant[profile].user_transports[]`|`ERROR: set command for Participant <user_transports> expects 1 arguments and received 0`|`PARTICIPANT_USER_TRANSPORTS_USAGE`|
 |`fastddsqosprof file.xml set participant[profile].user_transports[].subelement`|`ERROR: Participant <user_transports> must be FINAL element`|`PARTICIPANT_USER_TRANSPORTS_USAGE`|
 |`fastddsqosprof file.xml set participant[profile].user_transports[] -h`|N/A|`PARTICIPANT_USER_TRANSPORTS_USAGE`|
 |`fastddsqosprof file.xml set participant[profile].user_transports[] --help`|N/A|`PARTICIPANT_USER_TRANSPORTS_USAGE`|
 |`fastddsqosprof file.xml set participant[profile].user_transports[] help`|N/A|`PARTICIPANT_USER_TRANSPORTS_USAGE`|
 |`fastddsqosprof file.xml set participant[profile].user_transports[] arg1 arg2 -h`|N/A|`PARTICIPANT_USER_TRANSPORTS_USAGE`|
 |`fastddsqosprof file.xml set participant[profile].user_transports[] arg1 arg2`|`ERROR: set command for Participant <user_transports> expects 1 arguments and received 2`|`PARTICIPANT_USER_TRANSPORTS_USAGE`|
-|`fastddsqosprof file.xml set participant[profile].user_transports[] argument`|N/A|N/A|
-|`fastddsqosprof file.xml set participant[profile].user_transports[0] argument`|N/A|N/A|
+|`fastddsqosprof file.xml set participant[profile].user_transports[] arg1`|N/A|N/A|
+|`fastddsqosprof file.xml set participant[profile].user_transports[0] arg1`|N/A|N/A|
```

Comment:
My suggestion here was to have first one value `argument` and then update that value to a different one `arg1`.